doc/development/deprecations.rst

```diff
@@ -73,6 +73,11 @@ for details on how to port your legacy code to the new system. The following fun
 Completed deprecation cycles
 ----------------------------
 
+* The input argument ``control_wires`` of ``MultiControlledX`` has been removed.
+
+  - Deprecated in v0.22
```

Comment:
Note: we got the version info via private communication offline. Anyways it's a super far back time so don't worry too much about the accuracy.